app/src/main/java/com/android/feedme/ui/component/SmallCommentsDisplay.kt

```diff
@@ -61,7 +78,8 @@ fun CommentCard(comment: Comment) {
       shape = RoundedCornerShape(8.dp),
       border = BorderStroke(2.dp, Color.Black)) {
         Row(modifier = Modifier.fillMaxWidth().padding(16.dp)) {
-          // Recipe image
+
+          // TODO replace this by the photo uploading code
```

Comment:
I am unsure what the TODO is for, why would you need to upload a photo on the display of the comment ?

Reply:
You're right! It's totally not supposed to be here ^^' Thanks for the spotting, I'll remove it (: